.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please remove all .vs folders.

(I will add gitignore as well)

Reply:
@yangyuan, Done. Thanks! Can you please check?

Reply:
On .vs is still there... in the root folder :) 
/.vs/